cl/corpus_importer/management/commands/normalize_judges_opinions.py

```diff
@@ -0,0 +1,88 @@
+from juriscraper.lib.string_utils import titlecase
+
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.people_db.lookup_utils import (
+    extract_judge_last_name,
+    lookup_judge_by_last_name,
+    lookup_judges_by_last_name_list,
+)
+from cl.search.models import Opinion, OpinionCluster
+
+
+def normalize_authors_in_opinions():
+    """Normalize author_str in opinions"""
+
+    # Get opinions that have no author object and have author_str
+    opinions_with_author_str = Opinion.objects.exclude(
+        author__isnull=False
+    ).exclude(author_str="")
+    for opinion in opinions_with_author_str.iterator():
+        date_filed = opinion.cluster.docket.date_filed
+        court_id = opinion.cluster.docket.court_id
+        # Search for person, living or deceased
+        person = lookup_judge_by_last_name(
+            opinion.author_str, court_id, date_filed, False
+        )
+
+        if not person:
+            logger.warning(
+                f"Can't find person with this last name: "
+                f"{opinion.author_str} in opinion id: {opinion.pk}"
+            )
+        else:
+            # Set person object to author
+            opinion.author = person
+            opinion.save()
+            logger.info(f"Author updated in opinion id: {opinion.pk}")
+
+
+def normalize_panel_in_opinioncluster():
+    """Normalize panel in opinion cluster object"""
+
+    # Get opinion cluster that have no panel objects and have judges_str
+    opinion_clusters_with_judges_str = OpinionCluster.objects.exclude(
+        panel__isnull=False
+    ).exclude(judges__exact="")
```

Comment:
I don't think there's almost any reason to ever use `__exact`. Let's remove it here too:

```suggestion
    ).exclude(judges="")
```